ui-automated-tests/src/utils.ts

```diff
@@ -129,6 +129,17 @@ export const matchText = async (tag: string, text: string, quote?: string) => {
     .equal(1, `No text found using '//${tag}[contains(text(), ${quote}${text}${quote})]'`);
 };
 
+export const matchText2 = async (tag: string, text: string, quote?: string) => {
```

Comment:
I'm not a fan of this function name, but I'm struggling to explain why I needed to do this (the XPath pattern fed to`this.getElementHandles()` below was required and the one being used in `matchText()` would not work, for reasons I can't really explain).

Reply:
https://sqa.stackexchange.com/questions/36918/xpath-text-buy-now-is-working-but-not-containstext-buy-now